src/s3_upload.py

```diff
@@ -62,6 +62,9 @@ def upload_todays_events_to_s3():
     end_time = time.time()
     logger.info(f"Uploaded {len(files_updated_today)} files to s3, took {end_time - start_time} seconds.")
 
+    # cleanup old files, free up disk space
+    cleanup_old_files()
```

Comment:
You could pass `start_time` in as an argument if you want to delete more files when a deploy is going as the day rolls over